Sources/VariadicsGenerator/VariadicsGenerator.swift

```diff
@@ -123,6 +123,12 @@ struct VariadicsGenerator: ParsableCommand {
       import _MatchingEngine
 
 
+      private func concat(
```

Comment:
It would be simpler if we change code generation to output `.concatenation(...)` instead of `concat(...)`.